nixos/modules/services/web-apps/kimai.nix

```diff
@@ -0,0 +1,218 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
```

Comment:
```suggestion
```

Reply:
something wrong `with lib`?

Reply:
It makes difficult for anyone to immediately see what's being used from `lib`.

Instead use something like
```nix
let
  inherit (lib) mkOptions ...;
in
  ...
```